### ISSUE TITLE
Can't find source repository if the fork has a different name than the repo

### ISSUE THREAD

**Issue by athei** [paritytech/bench-bot#86]
Error happened here: https://github.com/paritytech/substrate/pull/10789#issuecomment-1029969372

The issue seems to be that the fork where the PR was made from has a different name than the repo where the bot resides on.

**Comment by shawntabrizi** [comment]
No, I don't think that is a common scenario, but supporting forks in general def is.
